File: GTTS/INL.VettingService.Models/SavePersonVettingVettingType_Param.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class SavePersonVettingVettingType_Param : ISavePersonVettingVettingType_Param
    {
        public long? PersonVettingID { get; set; }
        public int? VettingTypeID { get; set; }
        public bool? CourtesySkippedFlag { get; set; }
        public string CourtesySkippedComments { get; set; }
        public int? ModifiedAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/save-person-vetting-vetting-type_param.ts


import { ISavePersonVettingVettingType_Param } from './isave-person-vetting-vetting-type_param';

export class SavePersonVettingVettingType_Param implements ISavePersonVettingVettingType_Param {
  
	public PersonVettingID?: number;
	public VettingTypeID?: number;
	public CourtesySkippedFlag?: boolean;
	public CourtesySkippedComments: string = "";
	public ModifiedAppUserID?: number;
  
}


